src/utils/streams-service.js

```diff
@@ -0,0 +1,36 @@
+/**
+ * Copyright (c) [2020] SUSE Linux
+ *
+ * This software may be modified and distributed under the terms
+ * of the MIT license.  See the LICENSE.txt file for details.
+ */
+
+export default (function() {
```

Comment:
I have not managed to make it work as you suggest. Can you please just add a commit correcting it?

Reply:
> I have not managed to make it work as you suggest.

My fault, since I put a not accurate example. I simply mixed some things :)

Anyway, It's already done and even merged :smiley: